src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/configuration/configuration.component.spec.ts

```diff
@@ -35,4 +36,13 @@ describe('ConfigurationComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should check header text', () => {
```

Comment:
Thanks for adding the unit test. Here is a diff how you can greatly reduce this tests.

```diff
diff --git a/src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/configuration/configuration.component.spec.ts b/src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/configuration/configuration.component.spec.ts
index 5d62513aa7..712c478dab 100644
--- a/src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/configuration/configuration.component.spec.ts
+++ b/src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/configuration/configuration.component.spec.ts
@@ -38,11 +38,8 @@ describe('ConfigurationComponent', () => {
   });
 
   it('should check header text', () => {
-    const dataTableHeader = fixture.debugElement.query(By.css('.datatable-header'));
-    expect(dataTableHeader.nativeElement.textContent).toContain('Name');
-    expect(dataTableHeader.nativeElement.textContent).toContain('Description');
-    expect(dataTableHeader.nativeElement.textContent).toContain('Current value');
-    expect(dataTableHeader.nativeElement.textContent).toContain('Default');
-    expect(dataTableHeader.nativeElement.textContent).toContain('Editable');
+    expect(fixture.debugElement.query(By.css('.datatable-header')).nativeElement.textContent).toBe(
+      ['Name', 'Description', 'Current value', 'Default', 'Editable'].join('')
+    );
   });
 });
```